identifiers/country-mx/districts.csv

```diff
@@ -0,0 +1,300 @@
+ocd-division/country:mx/state:gr/fed:3,III Federal Electoral District of Guerrero
+ocd-division/country:mx/state:gr/fed:4,IV Federal Electoral District of Guerrero
+ocd-division/country:mx/state:gr/fed:5,V Federal Electoral District of Guerrero
+ocd-division/country:mx/state:gr/fed:6,VI Federal Electoral District of Guerrero
+ocd-division/country:mx/state:gr/fed:7,VII Federal Electoral District of Guerrero
+ocd-division/country:mx/state:gr/fed:8,VIII Federal Electoral District of Guerrero
+ocd-division/country:mx/state:gr/fed:9,IX Federal Electoral District of Guerrero
+ocd-division/country:mx/state:hg/fed:1,I Federal Electoral District of Hidalgo
+ocd-division/country:mx/state:hg/fed:2,II Federal Electoral District of Hidalgo
+ocd-division/country:mx/state:hg/fed:3,III Federal Electoral District of Hidalgo
+ocd-division/country:mx/state:hg/fed:4,IV Federal Electoral District of Hidalgo
+ocd-division/country:mx/state:hg/fed:5,V Federal Electoral District of Hidalgo
+ocd-division/country:mx/state:hg/fed:6,VI Federal Electoral District of Hidalgo
+ocd-division/country:mx/state:hg/fed:7,VII Federal Electoral District of Hidalgo
+ocd-division/country:mx/state:ja/fed:1,I Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:2,II Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:3,III Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:4,IV Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:5,V Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:6,VI Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:7,VII Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:8,VIII Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:9,IX Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:10,X Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:11,XI Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:12,XII Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:13,XIII Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:14,XIV Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:15,XV Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:16,XVI Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:17,XVII Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:18,XVIII Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:19,XIX Federal Electoral District of Jalisco
+ocd-division/country:mx/state:ja/fed:20,XX Federal Electoral District of Jalisco
+ocd-division/country:mx/state:em/fed:1,I Federal Electoral District of Mexico
+ocd-division/country:mx/state:em/fed:2,II Federal Electoral District of Mexico
```

Comment:
This is listed as "México" on the INE site, would it be better to match?

Reply:
In the context of our discussions on language(s), English sounds perfect in this case!